plugins/llm-wiki/tests/resolver_smoke.sh

```
@@ -0,0 +1,32 @@
+CHK="skills/bootstrap-wiki/assets/templates"
+fail=0; pass(){ echo "  PASS: $1"; }; die(){ echo "  FAIL: $1"; fail=1; }
+
+RES=$(awk '/# --- Plugin root resolution/{f=1} f{print} /plugin root not resolved/{if(f)exit}' "$SKILL" | sed 's/^[[:space:]]*//')
+[ -n "$RES" ] || { echo "  FAIL: could not extract resolver from $SKILL"; exit 1; }
+
+TMP=$(mktemp -d); trap 'rm -rf "$TMP"' EXIT
+mkdir -p "$TMP/cache/mkt/llm-wiki" "$TMP/foreign"
+ln -s "$PLUGIN_DIR" "$TMP/cache/mkt/llm-wiki/2.5.0"
+
+# Codex: no CLAUDE_PLUGIN_ROOT, foreign cwd, fake plugin cache
+OUT=$(cd "$TMP/foreign" && env -u CLAUDE_PLUGIN_ROOT -u HERMES_HOME CODEX_PLUGIN_CACHE="$TMP/cache" bash -c "$RES"$'\n''echo "$PLUGIN_ROOT"')
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)